client/registry.go

```diff
@@ -27,11 +27,11 @@ import (
 	"strings"
 )
 
-func (c *Client) AspectGet(aspectID string, requests []string) (result map[string]interface{}, err error) {
+func (c *Client) GetThrough(registryView string, requests []string) (result map[string]interface{}, err error) {
```

Comment:
I'm not sure I really understand the usage of `Through` in the contexts that we're using it.

Reply:
I interpreted it as getting a value through a specific view. I thought of suggesting `GetThroughView` or `GetThroughRegistryView` but I'm not sure they're much better?

Reply:
Also discussed here https://github.com/snapcore/snapd/pull/14050#discussion_r1634486460

Reply:
I meant GetThroughView to be clear, that what the ... for, sorry if it was confusing

Reply:
get through alone is a phrasal verb that doesn't apply here

Reply:
Ah my bad, I assumed that was for arguments. Will correct this

Reply:
@andrewphelpsj @MiguelPires maybe GetViaView  that is shorter?

Reply:
Then registrystate Set should perhaps only be `SetThroughView` since registry is part of the package?

Reply:
I think in client we should use fulll names RegistryGetViaView ?  but the packages that have registry in the name we should skip the registry bit whenever possible?